packages/block-library/src/navigation/edit/index.js

```diff
@@ -237,6 +209,40 @@ function Navigation( {
 		[ navigationMenus ]
 	);
 
+	// This useEffect adds snackbar and speak status notices when menus are created.
+	// If there are no fallback navigation menus then we don't show these messages,
+	// because this means that we are creating the first, fallback navigation menu.
```

Comment:
Can we / Should we  - also hide the snackbar about the importing of the default classic menu? If I have a classic menu and it is used as the basis for the default block navigation I get a snackbar saying classic menu imported ..

Reply:
I thought about that, but I'm not sure. It's a different case and I thought it might be useful to know what's happened in that case.